cfme/tests/storage/test_volume.py

```diff
@@ -46,7 +51,48 @@ def volume(appliance, provider):
             msg=str(e)))
 
 
-def test_storage_volume_create_cancelled_validation(appliance, provider):
+@pytest.fixture(scope="function")
+def instance_fixture(appliance, provider, small_template):
+    instance = appliance.collections.cloud_instances.instantiate(random_vm_name('stor'),
+                                                                 provider,
+                                                                 small_template.name)
+    if not instance.exists_on_provider:
+        instance.create_on_provider(allow_skip="default", find_in_cfme=True)
+    elif instance.provider.one_of(EC2Provider) and instance.mgmt.state == VmState.DELETED:
+        instance.mgmt.rename('test_terminated_{}'.format(fauxfactory.gen_alphanumeric(8)))
```

Comment:
Q: why you need to rename this?

Reply:
It's for not to have duplicate volume name. It's used in methods across all the tests.